### PR TITLE
Filter elt.parentNode Google Search app bug

### PR DESCRIPTION
**Pull request by benvinegar**
See: https://github.com/getsentry/raven-js/issues/756

cc @dcramer @lewisjellis

### CODE REVIEW

**Comment by benvinegar** [comment]
So, this might feel a little broad (what if someone has `elt.parentNode` that is breaking)? But the error string is a Safari error string. The same error on Chrome/Chromium generates a completely different message:

```
Uncaught TypeError: Cannot read property 'parentNode' of null
```

So, if this became a legit bug, people would definitely see it.

File: tests/sentry/filters/test_browser_extensions.py [hunk 1]
```diff
@@ -43,6 +43,10 @@ def test_filters_conduit_toolbar(self):
         data = self.get_mock_data(exc_value='what does conduitPage even do')
         assert self.apply_filter(data)
 
+    def test_filters_google_search_app_ios(self):
```
Comment (benvinegar) [inline tests/sentry/filters/test_browser_extensions.py]: This test is probably worthless, but hey.

**Review by LewisJEllis — APPROVED** [review]
LGTM, makes sense from skimming issue discussion

**Comment by Turbo87** [comment]
@benvinegar it seems we just hit the issue you described above. We are seeing errors like this now:

```
TypeError: Cannot read property 'parentNode' of null
  at ? (/)
```

**Pull request merged**